docs/rest_api.md

```diff
@@ -80,11 +80,14 @@ wsk namespace list -v
 ```
 ```
 REQUEST:
-[GET]	https://openwhisk.ng.bluemix.net/api/v1/namespaces
+[GET]  https://openwhisk.ng.bluemix.net/api/v1/namespaces
 Req Headers
 {
   "Authorization": [
     "Basic XXXYYYY"
+  ],
+  "User-Agent": [
```

Comment:
this is an example of a cli command with the `-v` which displays request header.  when the cli inserts its own agent-header, this will now appear in the `-v` output.. so, this update looks good.  was there something else you had in mind @rabbah ?